### PR TITLE
Remove font awesome

### LINKED ISSUE #12: Remove Font Awesome Dependency
Font awesome dependency can be removed with the usage of predefined inline svg with the help of the [simple-icons package](https://github.com/simple-icons/simple-icons/tree/develop/icons).

For instance, [network property, Facebook](https://github.com/ozdemirburak/jquery-floating-social-share/blob/master/src/jquery.floating-social-share.js#L89) can be extended like below where Facebook SVG can be found [here](https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/facebook.svg).

```js
"facebook" : {
  className: "facebook",
  icon: "<svg aria-labelledby='simpleicons-facebook-icon' role='img' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><title id='simpleicons-facebook-icon'>Facebook icon</title><path d='M22.676 0H1.324C.593 0 0 .593 0 1.324v21.352C0 23.408.593 24 1.324 24h11.494v-9.294H9.689v-3.621h3.129V8.41c0-3.099 1.894-4.785 4.659-4.785 1.325 0 2.464.097 2.796.141v3.24h-1.921c-1.5 0-1.792.721-1.792 1.771v2.311h3.584l-.465 3.63H16.56V24h6.115c.733 0 1.325-.592 1.325-1.324V1.324C24 .593 23.408 0 22.676 0'/></svg>",
  url:"https://www.facebook.com/sharer/sharer.php?u={url}&t={title}"
},
```

Then just [modify the source code](https://github.com/ozdemirburak/jquery-floating-social-share/blob/master/src/jquery.floating-social-share.js#L45) that it will use the icon property or whatever instead of the font awesome one.

### PR DESCRIPTION
**Pull request by marioidival**
Fixes #12

### CODE REVIEW

**Comment by ozdemirburak** [comment]
Ermm, this should not just remove the font awesome dependency, instead, should define the icons for each network and use them.

**Comment by marioidival** [comment]
Oh, Ok! ^^

**Comment by marioidival** [comment]
@ozdemirburak done

File: src/jquery.floating-social-share.js [hunk 1]
```diff
@@ -42,14 +42,13 @@
       $.each(this.settings.buttons, function(index, value) {
         $.each(networks, function(k, v) {
           if (value === k) {
-            var $icon = $("<i>", { class: "margin-top-5 fa fa-" + value }),
```
Comment (ozdemirburak) [inline src/jquery.floating-social-share.js]: This should be equal to the SVG defined. Something like below.

`var $icon = $(networks[value]['icon']),`

File: src/jquery.floating-social-share.js [hunk 2]
```diff
@@ -42,14 +42,13 @@
...
-              _href = v.url.replace('{url}', encodeURIComponent(base.settings.url))
+            var _href = v.url.replace('{url}', encodeURIComponent(base.settings.url))
                 .replace('{title}', encodeURIComponent(base.settings.title))
                 .replace('{description}', encodeURIComponent(base.settings.description))
                 .replace('{media}', encodeURIComponent(base.settings.media)),
               _text_value = base.settings.text[value] || _text_default + value,
               _text_output = base.settings.text_title_case ? title_case(_text_value) : _text_value,
-              $component = $("<a>", { title: base.settings.title, class: v.className + " pop-upper"}).attr("href", _href).attr("title", _text_output).append($icon);
+              $component = $("<a>", { title: base.settings.title, class: v.className + " pop-upper"}).attr("href", _href).attr("title", _text_output);
```
Comment (ozdemirburak) [inline src/jquery.floating-social-share.js]: No need to change this line.

**Comment by ozdemirburak** [comment]
Will merge after you changed those back :)

File: src/jquery.floating-social-share.js (hunk 2 above)
Comment (marioidival) [reply src/jquery.floating-social-share.js]: Done

File: src/jquery.floating-social-share.js (hunk 1 above)
Comment (marioidival) [reply src/jquery.floating-social-share.js]: Done

**Comment by marioidival** [comment]
captura de tela 2017-10-02 as 08 47 31

**Comment by ozdemirburak** [comment]
Thanks!

**Pull request merged**